File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L3/08/Level3.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let berat: number;
let feet: number;
let inches: number;
let tinggi: number;
let bmi: number;

rl.question('Enter weight in pounds: ', (weightInput) => {
  berat = parseFloat(weightInput);
  rl.question('Enter feet: ', (feetInput) => {
    feet = parseFloat(feetInput);
    rl.question('Enter inches: ', (inchesInput) => {
      inches = parseFloat(inchesInput);
      tinggi = feet * 12 + inches;
      bmi = berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254));
      console.log('BMI is ' + bmi.toFixed(2));
      if (bmi < 18.5) {
        console.log('Underweight');
      } else if (bmi < 25) {
        console.log('Normal');
      } else if (bmi < 30) {
        console.log('Overweight');
      } else {
        console.log('Obese');
      }
      rl.close();
    });
  });
});

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L3/08/Level3.kt
package IR

import java.util.Scanner

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level3 {
    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)
        var berat: Double
        var feet: Double
        var inches: Double
        var tinggi: Double
        var bmi: Double

        print("Enter weight in pounds: ")
        berat = `in`.nextDouble()
        print("Enter feet: ")
        feet = `in`.nextDouble()
        print("Enter inches: ")
        inches = `in`.nextDouble()
        tinggi = feet * 12 + inches

        bmi = berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254))

        println("BMI is $bmi")
        when {
            bmi < 18.5 -> println("Underweight")
            bmi < 25 -> println("Normal")
            bmi < 30 -> println("Overweight")
            else -> println("Obese")
        }
    }
}

